src/workspace/Lesson/SectionOverview/Settings/index.jsx

```diff
@@ -8,16 +8,17 @@ import Flex from "~/components/Flex"
 
 import previewIcon from "~/assets/preview-white.svg"
 import noPreviewIcon from "~/assets/no-preview-white.svg"
-import trashIcon from "~/assets/trash-white.svg"
+import TrashIcon from "~/assets/trash-white.svg"
 import duplicateIcon from "~/assets/duplicate-white.svg"
 import shareIcon from "~/assets/share-white.svg"
 import infoIcon from "~/assets/info-white.svg"
-import closeIcon from "~/assets/close-white.svg"
+import CloseIcon from "~/assets/close-white.svg"
```

Comment:
Wenn dann als Komponente TrashIcon, CloseIcon. Wir können auch gerne für die svgs ohne weiteren Inhalt das so lösen, wie in vue. Das wir dann über CustomButton name = "close-white" gehen. Auch für die anderen Stellen, die nur svgs importieren und direkt verwenden.

Reply:
Das sind quasi Komponententen. SVGR convertiert svgs als react componenten

Reply:
https://ticketsystem.schul-cloud.org/browse/SC-3478